cmd-info.c

```diff
@@ -779,11 +779,6 @@ int command_info(int argc, char *argv[], struct opts *opts)
 	if (ret < 0)
 		pr_warn("cannot open data: %s\n", opts->dirname);
 
-	snprintf(buf, sizeof(buf), "%s/info", opts->dirname);
-
-	if (stat(buf, &statbuf) < 0)
-		return -1;
```

Comment:
Hmm..?  It needs statbuf to know the time it recorded.  But I also think it'd be better passing the timestamp (+ elapsed  itme maybe) as a separate info item.